test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -35,6 +35,43 @@ Describe "Get-ChildItem" -Tags "CI" {
             $files[2].Name     | Should Be "B"
             $files[3].Name     | Should Be "c"
             $files[4].Name     | Should Be "D"
+            # Test data cleanup
+            Remove-Item -Path $TestDrive -Recurse -Force
```

Comment:
There's no need to remove TestDrive. Pester internally takes care of that. So, removing this statement.